tests/config/study/profit.yaml

```diff
@@ -0,0 +1,16 @@
+ntrain: 2
+variables:
+    u: Halton()
```

Comment:
Shouldn't Uniform and Halton do the same thing? Otherwise, is it possible to add ranges also for Halton to rescale?